lib/index.js

```diff
@@ -268,19 +268,20 @@ class Mcdev {
      * @param {string} cred name of Credential
      * @param {string} bu name of BU
      * @param {string[]} [typeArr] limit deployment to given metadata type
+     * @param {string[]} [keyArr] limit deployment to given metadata keys
      * @returns {Promise} ensure that BUs are worked on sequentially
      */
-    static async _deployBU(cred, bu, typeArr) {
+    static async _deployBU(cred, bu, typeArr, keyArr) {
```

Comment:
how does this deal with a scenario where there is multiple types and multiple keys?